config/api.yml

```diff
@@ -117,6 +117,17 @@
     :klass: ChargebackRate
     :subcollections:
     - :rates
+  :container_deployments:
+    :description: Container Provider Deployment
+    :identifier: container_deployment
+    :klass: ContainerDeployment
+    :methods: *70174834084700
+    :options:
+    - :collection
```

Comment:
@abellotti I had to remove all the read methods to make it work(https://github.com/ManageIQ/manageiq/pull/9231/files#diff-15502b585d5033aa8043cc26237a06baR166), how come this fails here and on master it doesn't?